backend/src/services/referral.ts

```diff
@@ -33,11 +36,39 @@ export async function editReferral(
   assignedReferringStaffId: string,
   status: string,
 ) {
+  const Ref = await ReferralModel.findById(id);
+  const updateHL = Ref?.assignedHousingLocator?.toString() !== assignedHousingLocatorId;
+  const setLeased =
+    status === "leased" &&
```

Comment:
```suggestion
    status === "Leased" &&
```
Should this be `"Leased"` if we are expecting all statuses to start with a capital letter?